src/tests/fuzz/README

```diff
@@ -0,0 +1,11 @@
+How to run fuzzing ?
```

Comment:
how do I control which Suricata repo+branch is used?

Reply:
This is in `Dockerfile` used by oss-fuzz
See https://github.com/google/oss-fuzz/pull/2533/files#diff-f132db5f0e91b1506efa943dd4eb789bR30